examples/python/robot-wrapper-viewer.py

```diff
@@ -0,0 +1,69 @@
+##
+## Copyright (c) 2018 CNRS
+##
+## This file is part of Pinocchio
```

Comment:
This signature is deprecated and has to be removed.

Reply:
Uh, You mean, I should simply remove all the copyright-related comments on top of the file? I actually, I don't know how this one ended up here. I will remove it.

Reply:
You can keep the copyright, but you just need to remove the license.

Reply:
Ok